src/main/scala/me/shadaj/romeojuliet/Tragedy.scala

```diff
@@ -21,6 +21,53 @@ case class YouHaveToMarry(that: ActorRef)
 case object HelpMe
 
 case object SleepingPotion
+case class WakingUp(actorRef: ActorRef)
+
+object Tragedy {
+  final val Name = "tragedy"
+  def props: Props = Props(new Tragedy)
+}
+
+/* The single top-level actor. */
+class Tragedy extends Actor {
```

Comment:
Each actor system should have a single top-level actor. In Akka 3 (akka-typed) this is mandatory.